src/vuestic-theme/vuestic-components/va-checkbox/VaCheckbox.demo.vue

```diff
@@ -85,6 +92,7 @@ export default {
   data () {
     return {
       value: true,
+      selection: [],
```

Comment:
Does nothing.

Reply:
This is target array for demo purposes.
It passes to va-checkbox components set.
```
    <VbCard title="Array as model">
      {{selection}}
      <va-checkbox v-model="selection" array-value='one' label="one"/>
      <va-checkbox v-model="selection" array-value='two' label="two"/>
      <va-checkbox v-model="selection" array-value='three' label="three"/>
      <va-checkbox v-model="selection" array-value='four' label="four"/>
    </VbCard>
```